### ISSUE TITLE
[BUG] 

### ISSUE THREAD

**Issue by ashubabu** [microsoft/accessibility-insights-windows#1114]
**Describe the bug**
Accessibility behavior of MS Teams on Dual Monitor, with Monitors setup at different scales, example 100% and 125%, with 1920*1080 resolution. The tool I use is Accessibility Insight as well as I have tried to check the results with Automation API. Both show similar behavior.

From Task Manager, I see MS Teams Process, responsible for UI, is a Per Monitor DPI Aware App and so is Accessibility Insight, However, Accessibility Insight as well as Windows Automation API, hosted in a sample project (code snippet below) that I developed and that runs in Per Monitor DPI Aware mode, are unable to locate the MS Teams' Elements correctly in dual Monitor Configuration.

Further, MS Teams responds well on single Monitor with any scale factor or resolution. 

**To Reproduce**

1. Use Dual Monitors with Monitor 1 as Primary. Monitor 1 is set at 100%, 1920x1080 while Monitor 2 is set at 125%, 1920x1080. Both of Monitors are 14 inches which keeps Physical Screen Size out of equation.
2. Launch Teams App in Monitor 1 (100%) and move it to Monitor 2 (125%)
3. Open Accessibility Insight in Monitor 1 (or Monitor 2)
4. Try to locate "Search" box on Title Bar of Teams, where you can Search for a Person. You can try any other Element as well.
5. The Bounding Rectangle I see has Left as 2935 and Top as 280 while the actual Physical Coordinates that I get using GetPhysicalCursorPos (user32.dll) is X:2454 & Y:10
6. I wrote a Sample Project that hosts Windows Automation API and ran it in DPI Unaware, System DPI Aware and Per Monitor DPI Aware mode. In DPI Unaware Mode and System DPI Aware Mode, I get Bounding Rectangle Left as 2348 and Top as 10. If I scale the Bounding Rectangle Left value, obtained in step 5, down by 1.25 to account for Scaling (2935/1.25), I get 2348. Which makes me believe that scaling is done on presuming the Monitor is on 100%, probably because it was launched on Monitor 1 with 100% scale factor. 
7. When I run the Sample Project in Per Monitor DPI Aware mode, I get value Bounding Rectangle Left and Top as 2935 and 280, that matches with Accessibility Insight. 

**Expected behavior**
1. Considering Teams app runs in Per Monitor DPI Aware mode, I would expect the Accessibility Insight to give the correct location of the Element/s. In this case it should be Left ~2454 and Top ~10.

**Actual behavior**
1. Teams App doesn't seem to respond correctly to Monitor Scaling it seems. 

**Screenshots or .GIF**
The Screen Shot from the Custom App when Ran in Per Monitor DPI Aware mode.

![automationapi](https://user-images.githubusercontent.com/18587133/115676641-69fec880-a36d-11eb-940b-3c0f5d1c1234.PNG)

![sample code](https://user-images.githubusercontent.com/18587133/115677944-9d8e2280-a36e-11eb-81f6-9c8021c6ab05.PNG)

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10 1909] 
 - Accessibility Insights for Windows Version:
 - Target Application: [MS Teams]
 - Target Application Version: [e.g. 1.4.00.8872]

**Additional context**
If I enable GDI scaling, reading [Improve High DPIExperience](https://blogs.windows.com/windowsdeveloper/2017/05/19/improving-high-dpi-experience-gdi-based-desktop-apps/), I do see that Accessibility Insight is able to locate the Element as it should. Further, Accessibility Insight works well on "Display Settings" itself (SystemSettings.exe process), which is also Per Monitor Aware. I suspect that the response of the MS Teams Application is not on expected lines.

**Comment by waabid** [comment]
We've replied to the Stack Overflow question here: https://stackoverflow.com/questions/67189742/ms-teams-accessibility-insight-dual-monitor.

**Comment by ashubabu** [comment]
Thanks for your analysis and inputs. I did raise this as an issue yesterday itself with MS Teams Docs here - https://github.com/MicrosoftDocs/msteams-docs/issues/3154 since 

Will you be kind enough and inform me if there is any other Teams Repo/Channel where I should raise this concern? I could not find any Repository presence of MS Teams, understandably. I had raised it on MS Tech Community from where I was asked to raise this concern on github.

**Comment by DaveTryon** [comment]
Hi, @ashubabu! We think that our team has done all that we can on this issue, and that the issue should be closed out. If you agree, please close it out. Thanks!

**Comment by ashubabu** [comment]
Thanks for your inputs and efforts. I will follow it up with MS Teams. Closing it from here.
